azkaban-common/src/main/java/azkaban/executor/RunningExecutions.java

```diff
@@ -1,44 +1,41 @@
+/*
+ * Copyright 2018 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
 package azkaban.executor;
 
 import azkaban.utils.Pair;
-import java.util.Collection;
-import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 import javax.inject.Singleton;
 
+/**
+ * Provides access to running executions.
+ */
 @Singleton
 public class RunningExecutions {
```

Comment:
Is it necessary to wrap runningExecutions inside a new class?

Reply:
> Is it necessary to wrap runningExecutions inside a new class?

Not really, but this makes injection clearer. I could bind an instance of ConcurrentHashMap as a named singleton, but that would require using the `@Named` annotation also in the classes that require it in their `@Inject` constructor. So I would prefer having a class. Please let me know.